src/services/products.js

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import Utils from './utils';
+
+export async function getAllProducts() {
+    const config = {
+        headers: {
+          'Access-Control-Allow-Origin': '*',
+          'Accept': 'application/json',
+          'Authorization': localStorage.getItem('serverest/userToken'),
+        },
+    };
+
+    return await axios
+    .get(`${Utils.getBaseUrl()}/produtos`, config)
+    .then((response) => response)
+    .catch((error) => {
+        console.error(error.message);
+    })
+}
+
+export async function registerProduct(name, price, description, quantity) {
+    const config = {
+        headers: {
+          'Accept': 'application/json',
+          'Authorization': localStorage.getItem('serverest/userToken'),
+        }
+      };
+
+    return await axios
+      .post(`${Utils.getBaseUrl()}/produtos`,
+        {
+          nome: name,
+          preco: price,
+          descricao: description,
+          quantidade: quantity,
+      }, config)
+      .then((response) => response)
+      .catch((error) => error)
+}
+
+export async function registerProductWithImage(name, price, description, quantity, image) {
```

Comment:
```suggestion
export async function registerProductWithImage({ name, price, description, quantity, image }) {
```
Fazer o mesmo ajuste sugerido em `registerProduct`. É preciso alterar a forma que é consumido conforme sugeri em outro review.